vscode/src/debugger/activate.ts

```diff
@@ -70,10 +70,32 @@ function registerCommands(context: vscode.ExtensionContext) {
           { noDebug: true },
         ),
     ),
+    vscode.commands.registerCommand(
+      `${qsharpExtensionId}.runExpression`,
+      (expr: string) =>
+        startDebugging(
+          undefined,
+          {
+            name: "Run Expression",
+            stopOnEntry: false,
+            entry: expr,
+          },
+          { noDebug: true },
+        ),
+    ),
+    vscode.commands.registerCommand(
+      `${qsharpExtensionId}.debugExpression`,
+      (expr: string) =>
+        startDebugging(undefined, {
+          name: "Debug Expression",
+          stopOnEntry: true,
+          entry: expr,
+        }),
+    ),
   );
```

Comment:
I think I would have just updated the existing `debugEditorContents` and `runEditorContentsWithCircuit` commands with this new parameter. You won't have to update the existing call sites if you make the second parameter optional. Just to avoid proliferating command definitions. These are too similar to the old ones. 

I'm also surprised because I thought commands had to be defined in `package.json` but I don't see that change.